rest/route/patch_test.go

```diff
@@ -936,6 +940,11 @@ buildvariants:
 	assert.True(t, foundPassing)
 	assert.True(t, foundCompile)
 
+	// ensure that the patch contains all the tasks and didn't overwrite it with the new tasks
```

Comment:
nit: since the configure route is appending just `failing_test` to the existing ones, might be clearer to rephrase "contains all the tasks" to say the configured patch should include both the previously-scheduled and newly-scheduled tasks.